docs/release-notes/release-notes-0.20.0.md

```diff
@@ -25,6 +25,10 @@
 ## Functional Enhancements
 
 ## RPC Additions
+* When returning the response from [`ForwardingEvents`](https://github.com/lightningnetwork/lnd/pull/9813)
+, the response now include the incoming and outgoing htlc indices of the payment
+ circuit.
```

Comment:
We should mention that the indices are only available for the forwarding events saved after 0.20.